src/shared_modules/content_manager/src/components/CtiSnapshotDownloader.hpp

```diff
@@ -35,6 +35,9 @@ class CtiSnapshotDownloader final : public CtiDownloader
      */
     void download(UpdaterContext& context) override
     {
+        // Set the content type as snapshot.
+        context.data.at("type") = "raw";
```

Comment:
Delete this line, we dont use this class for snapshot download, this is for the mode cti-snapshot, and for offline we use the mode "offline"